src/components/home/sections/Volunteer.js

```diff
@@ -14,52 +9,24 @@ const SECTION_TITLE = 'An invitation from our board members'
 
 export const Volunteer = () => {
   return (
-    <section id='volunteer'>
-
-      <div className='volunteer-content'>
+    <div>
+      <section id='volunteer'>
 
-        <h1 className='volunteer-title'>{SECTION_TITLE}</h1>
-        <p>{CONTENT1}</p>
-        <a href='https://docs.google.com/forms/d/e/1FAIpQLScEOOuZ1iZw9IWbsiWc1AX3OHhFQ4wpxYs_TZHyykx2glsrFw/viewform' target='_blank' rel='noopener noreferrer'><button className='btn-container'>Sign Up</button></a>
-        <p>{CONTENT2}</p>
-        <p>{CONTENT3}</p>
+        <div className='volunteer-content'>
 
-        <a href='https://docs.google.com/forms/d/e/1FAIpQLSd9N1_wWETbeMs7UAxzUTCMl0FhCnVneozlA-EMAMQXSBFy3A/viewform' target='_blank' rel='noopener noreferrer'>
-          <button className='btn-container'>volunteer with us</button>
-        </a>
-      </div>
+          <h1 className='volunteer-title'>{SECTION_TITLE}</h1>
+          <p>{CONTENT1}</p>
+          <a href='https://docs.google.com/forms/d/e/1FAIpQLScEOOuZ1iZw9IWbsiWc1AX3OHhFQ4wpxYs_TZHyykx2glsrFw/viewform' target='_blank' rel='noopener noreferrer'><button className='btn-container'>Sign Up</button></a>
+          <p>{CONTENT2}</p>
+          <p>{CONTENT3}</p>
 
-      <div className='image-col'>
-        <div className='overlay'>
-          <img src={kim} alt='board-img' />
-          <div className='text-div'>
-            <p>Amber Kim</p>
-            <p>Chair</p>
-          </div>
+          <a href='https://docs.google.com/forms/d/e/1FAIpQLSd9N1_wWETbeMs7UAxzUTCMl0FhCnVneozlA-EMAMQXSBFy3A/viewform' target='_blank' rel='noopener noreferrer'>
+            <button className='btn-container'>volunteer with us</button>
+          </a>
         </div>
+      </section>
+      <BoardMembers />
```

Comment:
It would probably better to include this in the section as this whole part can be considered one section:
<img width="543" alt="Screen Shot 2020-11-08 at 1 15 35 PM" src="https://user-images.githubusercontent.com/14047327/98484473-8eeaef00-21c4-11eb-98fd-64c3fd12d0de.png">

The div that encompasses the paragraphs and the pictures can then be removed.

Reply:
Is it just me or does anyone else think the pics and words should be centered?

Reply:
@RaevLogic Are you talking about the text justification or the whole section being off? There is padding that's pushing and making the section off.

Reply:
![Untitled](https://user-images.githubusercontent.com/38801291/98484964-a6cc6e00-21d8-11eb-95fe-3ea33c5329e4.png)
I just edited it in Paint 3D real quick so it's not perfect, but that's more what I was thinking if we're splitting it in two sections like this.

Reply:
Two sections being the section with paragraphs and the section with all of our pictures.

Reply:
Ah. I'm good with that also.

Reply:
The only thing I don't like about this from a UI perspective is the split vision aspect. I find it awkward for our SPA to read down as a single column for the majority of the page, and then hit this portion and it's a two-column design where I have to read down the paragraphs and then go back up the page to visually scan back down the pictures section.

Reply:
🤷‍♀️I'm good with either one or two columns. I think if we have them stacked, I would put the pictures on top and then the text below.

Reply:
I think we can stick w/ the 2 column for now (unless it gets too annoying for you @rhtaylor ) and then create a new task for 1 column if we want to redesign to that.

Reply:
Lol, I'm not exactly a designer over here. Just letting everyone know my thoughts so I'm not wondering if I should have said something for two weeks xD

Reply:
Constructive feedback is always good. If it can improve user experience, we should think about it.